openrtb_ext/imp_flipp.go

```diff
@@ -1,9 +1,14 @@
 package openrtb_ext
 
+type ImpExtFlippOptions struct {
+	StartCompact *bool `json:"startCompact,omitempty"`
```

Comment:
Does it not make sense to make this options object generic? This way if there are more options that can be used in the future we don't need to update the adapter? We can update the documentation to mention what the supported options are.

Reply:
I think that came from first iteration of ticket, where the idea was to have smartCompact default to `true`. 
<img width="411" alt="Screenshot 2023-05-02 at 11 25 25" src="https://user-images.githubusercontent.com/48212785/235629844-6f06c70b-0c5f-4f92-acb3-bb5228e0fed4.png">

I'm happy to change to more generic approach, so we have less custom logic on the prebid server